util/exec.go

```diff
@@ -23,6 +23,7 @@ import (
 // from the binary's stdout and stderr to this program's stdout and stderr.
 func Exec(bin string, args ...string) error {
 	cmd := exec.Command(bin, args...)
```

Comment:
like, "gpg is required but not found - is it installed and on your PATH?" Something like that.